sql/src/main/java/io/crate/metadata/sys/ClassifiedMetrics.java

```diff
@@ -53,6 +54,7 @@
         Metrics(Classification classification, final long highestTrackableValue, final int numberOfSignificantValueDigits) {
             this.classification = classification;
             this.histogram = new ConcurrentHistogram(highestTrackableValue, numberOfSignificantValueDigits);
+            this.startTime = System.currentTimeMillis();
```

Comment:
should we expose this in the table?

Reply:
No, we want to get rid of the necessity for this in 4.0 by removing the pre-calculated metrics from the `QueryStats`